app.py

```diff
@@ -1,37 +1,127 @@
 from flask import Flask, render_template, request, redirect, url_for
-from item import TrelloItem
-from trello import TrelloClient
+from flask_login import LoginManager, login_required, login_user, current_user
+from oauthlib.oauth2 import WebApplicationClient
 from viewmodel import ViewModel
+from mongo_query import get_items, mark_item_as_complete, add_new_item, mark_item_as_uncomplete
+from user import User
+from userRole import UserRole
 import os
+import pymongo
+import requests
 
+def get_db_collection():
+
+    dbClientUri = os.getenv('COSMOS_MONGO_DB_CONNECTION')
+    databaseName = os.getenv('COSMOS_MONGO_DB_DATABASE')
+
+    collectionName = 'collection'
+
+    dbClient = pymongo.MongoClient(dbClientUri)
+    db = dbClient[databaseName]
+    return db[collectionName]
 
 def create_app():
     app = Flask(__name__)
 
-    client = TrelloClient(
-            os.getenv('API_KEY'),
-            os.getenv('TOKEN')
+    collection = get_db_collection()
+
+    app.secret_key = os.urandom(24)
+
+    login_disabled = os.getenv('LOGIN_DISABLED') == 'TRUE'
+    app.config['LOGIN_DISABLED'] = login_disabled
+    
+    login_manager = LoginManager()
+
+    client_id = os.getenv('AZURE_OAUTH_CLIENT_ID')
+    client_secret = os.getenv('AZURE_OAUTH_CLIENT_SECRET')
```

Comment:
These have changed name to `AZURE...` but haven't been updated in your template; and the client_id/secret doesn't actually have any bearing on Azure (We're still using the Github OAuth flow) so I'd be inclined to switch these back to CLIENT_ID/CLIENT_SECRET